### DISCUSSION TITLE
magick.exe command cannot recognize the absolute path in the Windows environment

### DISCUSSION THREAD

**Discussion by waverLv** [ImageMagick/ImageMagick#7045] [Help]
### ImageMagick version

7.1.1-26 Q8 x64

### Operating system, version and so on

windows11 X64

### Description

magick.exe: unable to open image 'D:\data\convert2html\39902309\0.jpg': No such file or directory @ error/blob.c/OpenBlob/3571.

**Comment by dlemstra** [comment]
I suspect that the file really doesn't exist. We can open files on Windows when we use a complete path.
